packages/js/components/src/experimental-tree-control/tree-item.scss

```diff
@@ -1,12 +1,21 @@
+$control-size: $gap-large;
+
 .experimental-woocommerce-tree-item {
 	margin: 0;
 
+	&--highlighted {
+		> .experimental-woocommerce-tree-item__heading {
+			background-color: #f6f7f7;
```

Comment:
If we ever implement dark mode I think we would have to go through all these "hard coded" values... But it's probably out of scope of this PR

Reply:
Good catch, it's done now. Thanks